### PR TITLE
Kickoff maquetado main

### PR DESCRIPTION
**Pull request by lucaspolonuer**

### CODE REVIEW

File: app/views/landing/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>Landing#index</h1>
+<p>Find me in app/views/landing/index.html.erb</p>
```
Comment (trueshizus) [inline app/views/landing/index.html.erb:2]: This file is an erb file, it should be a slim file

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+
+
+  get '/', to: 'landing#index', as: 'landing'
```
Comment (trueshizus) [inline config/routes.rb:4]: Are you sure `, as: 'landing'` is required?

**Review by trueshizus — CHANGES_REQUESTED** [review]

File: app/views/landing/index.html.erb (hunk 1 above)
Comment (lucaspolonuer) [reply app/views/landing/index.html.erb:2]: Yes, it was changed in the following branches, will be changing it here too

File: config/routes.rb (hunk 2 above)
Comment (lucaspolonuer) [reply config/routes.rb:4]: it provides a shortcut for accessing this path from anywhere in your code, not necessary now but could be useful later, you think I should remove it?

**Pull request closed without merging**